Safari.md

```diff
@@ -91,12 +91,12 @@ While using the IMAGE browser extension on Safari, please be aware of the follow
 
 - The extension does not produce any sound when an IMAGE request is sent.
 - The feedback link is missing when an IMAGE result is received for maps and charts.
-- The extension may become non-responsive and require a manual restart. [SATVIK: Can you provide more detail on what seems to trigger this, or how rare it is?]
+- The extension may become non-responsive and require a manual restart. This behavious is usually noticed when Safari is restarted in iOS.
```

Comment:
Typo "behavious". But more importantly, I'm still not sure what this means exactly. Are you saying that when Safari is restarted, that there is a chance the extension wil become unresponsive, and you'll have to reload it, even though Safari restarted? And it doesn't happen every time?

Reply:
let me correct it, it should be - when safari is restarted, extension will become unresponsive and will require a manual restart